src/main/java/Duke.java

```diff
@@ -1,10 +1,205 @@
+        item.setDone(false);
+        printResponse("    " + OperationsMessages.UNMARK_MSG + "\n" + "    " + item.toString());
+    }
+
+    private static void addItem(Todo item) {
+        itemList.add(item);
+        printResponse(String.format(OperationsMessages.ADDED_MSG, item.toString(), itemList.size()));
+    }
+
+    private static void removeItem(int itemIdx) {
+        itemIdx--;
+        Todo itemToDelete = itemList.get(itemIdx);
+        printResponse(String.format(OperationsMessages.DELETED_MSG, itemToDelete.toString(), itemList.size()));
+        itemList.remove(itemIdx);
+    }
+
+    private static void printItems() {
+        String response = OperationsMessages.SHOW_ITEMS_MSG + "\n";
+        for (int i = 0; i < itemList.size(); i++) {
+            Todo item = itemList.get(i);
+            response += String.format("    %d.%s\n", i+1, item.toString());
+        }
+        printResponse(response);
+    }
+
+    private static void writeToFile(String filePath, String content) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(content);
+        fw.close();
+    }
+
+    private static void save(String filePath) throws IOException {
+        // create directory and file if they dont exist
+        File fileObj = new File(filePath);
+        File dirObj = fileObj.getParentFile();
+        dirObj.mkdirs();
+        String saveString = "";
+        for (Todo item: itemList) {
+            saveString += item.getSaveRepresentation() + "\n";
+        }
+        writeToFile(filePath, saveString);
+    }
+
+    private static void load(String filePath) throws FileNotFoundException {
+        File loadFile = new File(filePath);
+        Scanner fileScanner = new Scanner(loadFile);
+        while(fileScanner.hasNext()) {
+            String line = fileScanner.nextLine();
+            String[] lineParts = line.split(" /// ");
+            String itemType = lineParts[0];
+            // load each item one by one
+            boolean isMarked = lineParts[1] == "1";
+            Todo newItem;
+            switch (itemType) {
+            case "T":
+                newItem = new Todo(lineParts[2]);
+                newItem.setDone(isMarked);
+                addItem(newItem);
+                break;
+            case "D":
+                newItem = new Deadline(lineParts[2], lineParts[3]);
+                newItem.setDone(isMarked);
+                addItem(newItem);
+                break;
+            case "E":
+                newItem = new Event(lineParts[2], lineParts[3], lineParts[4]);
+                newItem.setDone(isMarked);
+                addItem(newItem);
+                break;
+            default:
+                // TODO add exception
+                break;
+            }
+        }
+        fileScanner.close();
+    }
+
+    private static void printResponse(String response) {
+        System.out.println("    ____________________________________________________________");
```

Comment:
Code quality: Avoid magic strings

Consider extracting a constant for line divider so that it can be reused and will enhance readability.